app/views/response/response.html.erb

```diff
@@ -86,6 +87,21 @@ jQuery(document).ready(function() {
     <%= hidden_field_tag('return', @return) %>
     <%= hidden_field_tag  "isSubmit", :id=>"isSubmit" %>
 <% end %>
+
```

Comment:
This part actually is not related to giving peer-review (responses), but to initiating review_chats. So this may not be a good place. I you add a link on the "others' work" page, next to "begin" "edit" or "update" of each response map. In this way the chatting feature would be more visible.